### PR TITLE
Recognize kube version 1.21 as openshift 4.8

### PR DESCRIPTION
**Pull request by mmulholla**
Added Kube version 1.21 to be recognized as OpenShift version 4.8

Added test to confirm it works

### CODE REVIEW

**Review by baijum — APPROVED** [review]
LGTM

**Comment by mmulholla** [comment]
Spoke with @dperaza4dustbit - going to put a hold on this until 4.8 GA's

**Comment by mmulholla** [comment]
Converted to draft to prevent merge.

File: pkg/tool/oc_test.go [hunk 1]
```diff
@@ -2,23 +2,43 @@ package tool
...
+type fakeProcessExecutor48 struct {
+}
+
+var output48 = `
+serverVersion:
+  major: "1"
+  minor: "21"
+`
+
+func TestOcVersion48(t *testing.T) {
```
Comment (isutton) [inline pkg/tool/oc_test.go:37]: Perhaps those tests could be transformed into table tests?

File: pkg/tool/oc_test.go [hunk 2]
```diff
@@ -2,23 +2,43 @@ package tool
...
+func TestOcVersion47(t *testing.T) {
+	fp := fakeProcessExecutor{Output: output47}
 	oc := NewOc(fp)
 	version, _ := oc.GetVersion()
 	if version != "4.7.0" {
 		t.Error("Version mismatch", version)
 	}
 }
+
+type fakeProcessExecutor48 struct {
```
Comment (isutton) [inline pkg/tool/oc_test.go:28]: This type can be removed as it isn't being used.

Comment (mmulholla) [reply pkg/tool/oc_test.go]: removed it

File: pkg/tool/oc_test.go (hunk 1 above)
Comment (mmulholla) [reply pkg/tool/oc_test.go:34]: Will leave this for 4.9

**Pull request merged**